index.html

```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+                <li class="skill_container"><span class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><span class="skill">Ruby</span></li>
+            </ul>
+            <button class="project_link"><span class="link_text">See project</span></button>
+        </div>
+        <div class="project_container_4">
+            <div class="project_title">
+                <h3>Business Website</h3>
+            </div>
+            <div class="project_description">
+                <p>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. Has been the 
+                industry's standard.</p>
+            </div>
+            <div class="project_image"></div>
+            <ul class="project_skills">
+                <li class="skill_container"><span class="skill">HTML</span></li>
+                <li class="skill_container"><span class="skill">Bootstrap</span></li>
+                <li class="skill_container"><span class="skill">Ruby</span></li>
+            </ul>
+            <button class="project_link"><span class="link_text">See project</span></button>
+        </div>
+        <div class="project_container_5">
+            <div class="project_title">
+                <h3>Bank Website</h3>
+            </div>
+            <div class="project_description">
+                <p>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. Has been the 
+                industry's standard.</p>
+            </div>
+            <div class="project_image"></div>
+            <ul class="project_skills">
+                <li class="skill_container"><span class="skill">HTML</span></li>
+                <li class="skill_container"><span class="skill">Bootstrap</span></li>
+                <li class="skill_container"><span class="skill">Ruby</span></li>
+            </ul>
+            <button class="project_link"><span class="link_text">See project</span></button>
+        </div>
+        <div class="project_container_6">
+            <div class="project_title">
+                <h3>Trading Website</h3>
+            </div>
+            <div class="project_description">
+                <p>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. Has been the 
+                industry's standard.</p>
+            </div>
+            <div class="project_image"></div>
+            <ul class="project_skills">
+                <li class="skill_container"><span class="skill">HTML</span></li>
+                <li class="skill_container"><span class="skill">Bootstrap</span></li>
+                <li class="skill_container"><span class="skill">Ruby</span></li>
+            </ul>
+            <button class="project_link"><spsn class="link_text">See project</span></button>
+        </div>
+    </section>
+    <section class="about_me">
+        <div class="bio_header">
+            <h1>About me</h1>
```

Comment:
- [ ] [OPTIONAL] Kindly use another `hx` value for this heading. As best practice, an `h1` should not be more than one in a document. This confuses the major focus or central theme of the page - and this can have a negative effect on the SEO of the page

Reply:
@AbayomiOlaoye Thank you so much, I will get to it right away.